packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -333,7 +341,21 @@ test.describe('severity-2 #smoke', () => {
       );
 
       await signupReact.fillOutCodeForm(code);
-      await page.waitForURL(/products/);
+      /*
+       * We must `waitUntil: 'load'` due to redirects that occur here. Note,
+       * React signup for SubPlat has one additional redirect compared to Backbone.
+       * See notes in https://github.com/mozilla/fxa/pull/16078#issue-1993842384,
+       * we can look at this in the sunset content-server epic.
+       *
+       * React signup staging goes from:
```

Comment:
I went down a bit of a rabbit hole here since I was originally trying to fix this with a `timeout` but thankfully the page load appears to always pass - I left this comments in case it's helpful later.